WasatchNET/Spectrometer.cs

```diff
@@ -2843,6 +2869,9 @@ protected virtual double[] getSpectrumRaw(bool skipTrigger=false)
                     return null;
                 }
 
+                if (isInGaAs)

```

Comment:
Change requested: please add a check for FeatureMask.hardwareEvenOddCorrection as proposed in ENG-0034 Rev 1.12

Reply:
will do